src/ring_theory/adjoin_root.lean

```diff
@@ -97,6 +111,20 @@ instance [comm_semiring S] [comm_semiring K] [has_smul S K] [algebra S R] [algeb
   is_scalar_tower S K (adjoin_root f) :=
 submodule.quotient.is_scalar_tower _ _
 
+instance adjoin_root.is_scalar_tower_right [distrib_smul S R] [is_scalar_tower S R R] :
+  is_scalar_tower S (adjoin_root f) (adjoin_root f) :=
+⟨begin
+  rintro x ⟨y⟩ ⟨z⟩,
+  simp only [submodule.quotient.quot_mk_eq_mk, algebra.id.smul_eq_mul],
+  -- we need to copy this lemma for `ideal.quotient`; will make this proof far easier.
```

Comment:
Since `ideal.quotient` has already been ported, let's add a todo note for the port
```suggestion
    -- TODO (after the port)
    -- we need to copy this lemma for `ideal.quotient`; will make this proof far easier.
```